### ISSUE TITLE
reloadPage() throws error when a POST request was made.

### ISSUE THREAD

**Issue by macdabby** [Codeception/Codeception#2673]
Firefox is opening a modal "To display this page, Firefox must send information that will repeat any action (such as a search or order confirmation) that was performed earlier." to confirm resubmitting the form. This causes a failure.

**Comment by Naktibalda** [comment]
What error do you get in Codeception?

**Comment by macdabby** [comment]
strangely, I got two different errors:

 [Facebook\WebDriver\Exception\UnexpectedAlertOpenException] Unexpected modal dialog (text: To display this page, Firefox must send information that will repeat any action (such as a search or order confirmation) that was performed earlier.): To display this page, Firefox must send information that will repeat any action (such as a search or order confirmation) that was performed earlier.
Build info: version: '2.48.2', revision: '41bccdd', time: '2015-10-09 19:59:12'
System info: host: 'DABs-MacBook-Pro.local', ip: '10.0.1.4', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.11.2', java.version: '1.7.0_79'
*** Element info: {Using=css selector, value=body}
Session ID: 24e6809f-8d80-1748-9a1c-1fe6a81c1c58
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{platform=MAC, acceptSslCerts=true, javascriptEnabled=true, cssSelectorsEnabled=true, databaseEnabled=true, browserName=firefox, handlesAlerts=true, nativeEvents=false, webStorageEnabled=true, rotatable=false, locationContextEnabled=true, applicationCacheEnabled=true, takesScreenshot=true, version=43.0.2}]

And on the second run:

  [Facebook\WebDriver\Exception\NoSuchWindowException]
  Window not found. The browser window may have been closed.
  Build info: version: '2.48.2', revision: '41bccdd', time: '2015-10-09 19:59:12'
  System info: host: 'DABs-MacBook-Pro.local', ip: '10.0.1.4', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.11.2', java.version: '1.7.0_79'
  Driver info: driver.version: unknown 

I was able to get past it by sending:
        $this->pressKey('#quicksearchbox', WebDriverKeys::ENTER);
but I'm still testing to make sure it's going to work without having to put a wait in there. Also, I can't see any way to detect if that modal is open so I could send this key conditionally.
